### DISCUSSION TITLE
SelectAction select points (method fails to start)

### DISCUSSION THREAD

**Discussion by GiorgioMauro** [LayTec-AG/Plotly.Blazor#449] [Q&A] [answered]
Hi everyone,

I’ve encountered an issue where the **SelectedAction="ClickAction"** does not trigger when selecting a large number of points, as shown in the image below:
![image](https://github.com/user-attachments/assets/cb8a64f7-6488-4517-a4c7-7c3633d3712c)

Specifically, the method fails to start when more than 340 points are selected. Has anyone else experienced this problem? Is there a known solution or configuration adjustment that can resolve this?

Thank y'all in advance for your help!

**Comment by sean-mcl** [comment, marked as answer]
You probably use Blazor.Server?

Try adding this:

``` csharp
builder.Services.Configure<HubOptions>(o =>
{
    o.MaximumReceiveMessageSize = null;
});
```

**Reply by GiorgioMauro** [reply]
Yeah, my application is a blazor server.
I'm gonna try, gonna keep you posted

**Reply by GiorgioMauro** [reply]
hey @sean-mcl ,

I added your piece of code in my program.cs and it works, thank you m8
